### DISCUSSION TITLE
Emoji 14

### DISCUSSION THREAD

**Discussion by C1710** [C1710/blobmoji#91] [General]
![Surprise! It's "a bit" earlier this time](https://user-images.githubusercontent.com/10349490/125536838-046cc1a7-8ef6-4256-9e17-f7c2b967fe3a.jpg)

Again: Happy World Emoji Day!

(There are no breaking changes here except for full Emoji 14.0 (Beta) support)

<hr /><em>This discussion was created from the release <a href='https://github.com/C1710/blobmoji/releases/tag/v14.0'>Emoji 14</a>.</em>

**Comment by shreyasminocha** [comment]
Love the pictures that accompany the releases btw 😊

**Comment by choukawaii** [comment]
> ![Surprise! It's "a bit" earlier this time](https://user-images.githubusercontent.com/10349490/125536838-046cc1a7-8ef6-4256-9e17-f7c2b967fe3a.jpg)
> 
> Again: Happy World Emoji Day!
> 
> (There are no breaking changes here except for full Emoji 14.0 (Beta) support)
> 
> _This discussion was created from the release [Emoji 14](https://github.com/C1710/blobmoji/releases/tag/v14.0)._

It works on Windows 11. But yeah, silly me that I did test it first on Twitter web so I thought it didn't work at all. 🤣

**Reply by C1710** [reply]
I know that it works in general, but at least for me on Windows 10, the quality is really bad/pixelated.

Is it better for you on 11?
